src/lib/auth.ts

```diff
@@ -56,6 +56,15 @@ export const ROLES: {
 		'resin.device_environment_variable.all',
 		'resin.device_tag.all',
 		'resin.device_service_environment_variable.all',
+
+		// These are for the supervisor proxy
+		'resin.device.purge',
+		'resin.device.shutdown',
+
+		// These are for the actions proxy
```

Comment:
These may not be necessary to add here as I believe open-balena users do not have the actions proxy anyway. Will rebase on https://github.com/balena-io/open-balena-api/pull/1978 once merged, adding permissions here so we can more easily test https://github.com/balena-io/balena-api/pull/5655